assets/js/base/hocs/with-reviews.js

```diff
@@ -13,14 +13,18 @@ import { formatError } from '../utils/errors.js';
 
 const withReviews = ( OriginalComponent ) => {
 	class WrappedComponent extends Component {
-		constructor() {
-			super( ...arguments );
+		constructor( props ) {
+			super( props );
+
+			this.isPreview = Boolean( props.attributes.previewReviews );
```

Comment:
What about just doing `!! props.attributes.previewReviews` here?

Reply:
I usually prefer doing things easier for newcomers and I think `Boolean( ... )` is easier to understand than `!! ...`. But double negation seems to be a bit more performant, so I'm ok either way. Changed in 4e8dee42d0b980b22ba747ecff620f4bda387171.